larsim/EventGenerator/LightSource_module.cc

```diff
@@ -612,6 +614,22 @@ namespace evgen {
     return material ? (fMaterials.count(material->GetName()) > 0) : false;
   } // LightSource::MaterialPointFilter::accept()
 
+
+  //----------------------------------------------------------------------------
+  bool LightSource::readParametersFromInputFile() {
+    double x, y, z;
+    fInputFile >> x >> y >> z >> fT
+      >> fSigmaX >> fSigmaY >> fSigmaZ >> fSigmaT
+      >> fP >> fSigmaP >> fN;
+    fCenter = { x, y, z };
+    if (!fInputFile.good()) return false;
+
+    std::string dummy;
+    std::getline(fInputFile, dummy); // this can fail for what I care
```

Comment:
What is the extra getline call for?

Reply:
I am guessing here...
The format of the file being read includes one "parameter set" per line.
I imagine that the purpose was to discard any extra text (maybe comments?) after the parameter values.
The next call would then find the file ready at the beginning of the next line.

Reply:
Well, maybe someday we can clean up this us.  But this infelicity isn't worth holding up approval.